src/CqlBuilderPanel/ValueSets/Results/Results.tsx

```diff
@@ -179,23 +204,44 @@ export default function Results(props: ResultsProps) {
         header: "",
         accessorKey: "apply",
         cell: (row: any) => {
+          const valueSetDetails = table.getRow(row.cell.row.id).original;
           return (
-            <div className="inline-flex gap-x-2">
-              <button
-                className="action-button"
-                onClick={(e) => {
-                  handleOpen(row.cell.row.id, e);
+            <Box
+              sx={{
+                display: "flex",
+                flexDirection: "row",
+                gap: 1,
+                alignItems: "right",
+                justifyContent: "right",
+              }}
+            >
+              <ToolTippedIcon
+                tooltipMessage="Apply"
+                buttonProps={{
+                  "data-testid": `apply-valueset-${row.cell.row.id}`,
+                  "aria-label": `apply-valueset-${row.cell.row.id}`,
+                  size: "small",
+                  onClick: () => {
+                    setSelectedReferenceId(valueSetDetails.oid);
+                    setSelectedValueSetDetails(valueSetDetails);
+                    handleApplyValueSet(valueSetDetails);
+                  },
                 }}
-                tw="text-blue-600 hover:text-blue-900"
-                data-testid={`select-action-${row.cell.id}`}
-                aria-label={`select-action-${row.cell.id}`}
               >
-                <div className="action">Select</div>
-                <div className="chevron-container">
-                  <ExpandMoreIcon sx={{ color: "#0073c8" }} />
-                </div>
-              </button>
-            </div>
+                <ControlPointIcon color="primary" />
+              </ToolTippedIcon>
+              <Box sx={{ flexGrow: 1 }} />
+              <ActionCenter
+                canEdit={true}
```

Comment:
is this hardcoded because search is visible only for owned draft measures?

Reply:
yes, but also, I just realized I don't think I wired up canEdit within the action center component. So doing that now and adding a couple tests

Reply:
sounds good. Thanks Nate!